students/ramkumar_rajanbabu/lesson_01/assignment/test_integration.py

```diff
@@ -0,0 +1,47 @@
+"""Module for test integration for inventory management"""
+
+
+from unittest import TestCase
+from unittest.mock import patch
+import inventory_management.main as Main
+
+
+class ModuleTest(TestCase):
+    """Test Module"""
+
+    def test_add_items(self):
+        """Test add items"""
+        product_1 = [901, "Keyboard", "n", "n"]
```

Comment:
Your integration testing is not working. Check out your side effects and confirm they follow the right sequence.